### PR TITLE
[NOTEST][1LP][RFR] Added test coverage

### PR DESCRIPTION
**Pull request by ganeshhubale**
## Purpose or Intent
- Added manual test case

### CODE REVIEW

File: cfme/tests/automate/test_automate_manual.py [hunk 1]
```diff
@@ -767,3 +767,268 @@ def test_variable_pass_method_playbook():
...
+            7. Click on commit button. You will see flash message: "Error: Selected domain is
+               locked"
+            8. Then navigate to custom domain and unlock it
+            9. Perform step 5, 6 and 7(In this case, domain will get imported)
+            10. Go to custom domain
+        expectedResults:
+            1.
+            2.
+            3.
+            4. You should see existing as well as imported namespaces, classes, instances or methods
```
Comment (john-dupuy) [inline cfme/tests/automate/test_automate_manual.py]: Afaik I believe you need `expectedResults` 5-10 in order for this metadata to be parsed correctly.

File: cfme/tests/automate/test_automate_manual.py [hunk 2]
```diff
@@ -767,3 +767,268 @@ def test_variable_pass_method_playbook():
...
+
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: positive
+        casecomponent: Automate
+        testSteps:
+            1. "Automation-->Automate--> Customization-->Import/Export--> Click export without a
+               service dialog selected.
+            2. Do no The error ""At least 1 item must be selected for export"" appears as expected.
```
Comment (john-dupuy) [inline cfme/tests/automate/test_automate_manual.py]: Small typo `Do no`

File: cfme/tests/automate/test_automate_manual.py [hunk 3]
```diff
@@ -767,3 +767,248 @@ def test_variable_pass_method_playbook():
...
+            1.
+            2.
+            3. Datastores exported on local system in zip format
+            4.
+            5.
+            6.
+            7. You should see flash message: "Error: Selected domain is locked"
+            8.
+            9. Selected domain imported successfully
+            10. You should see existing as well as imported namespace, classe, instance or method
```
Comment (sshveta) [inline cfme/tests/automate/test_automate_manual.py]: Small typo : class

File: cfme/tests/automate/test_automate_manual.py [hunk 4]
```diff
@@ -767,3 +767,248 @@ def test_variable_pass_method_playbook():
...
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: positive
+        casecomponent: Automate
+        testSteps:
+            1. Create domain, namespace, class and instance pointing to method
+            2. Navigate to automate > automation > simulation page
+            3. Fill values for attribute/value pairs of namespace, class, instance and add message
+               attribute with any value and click on submit.
+            4. See automation.logs
```
Comment (john-dupuy) [inline cfme/tests/automate/test_automate_manual.py]: Small typo 
```suggestion
            4. See automation.log
```

File: cfme/tests/automate/test_automate_manual.py [hunk 5]
```diff
@@ -767,3 +767,248 @@ def test_variable_pass_method_playbook():
...
+            2.
+            3.
+            4. Description and enabled status of existing domain should update.
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(coverage=[1753586])
+@pytest.mark.customer_scenario
```
Comment (mshriver) [inline cfme/tests/automate/test_automate_manual.py]: :tada:

**Pull request merged**